wgsl/index.bs

```diff
@@ -199,9 +199,10 @@ Issue: (dneto) also lifetime.
 There are multiple levels of scoping depending on how and where things are
 declared.
 
-A declaration must not introduce a name when that name is already in scope at the start
+A declaration must not introduce a name when that name is already in the current scope at the start
```

Comment:
There is no global accounting of scopes, so "the current scope" is not well defined.

See https://github.com/gpuweb/gpuweb/pull/1511 (sorry for the implicit conflict).

Reply:
Should have explained more...
#1511 says that the scope of a declared identifier is the range of source text over which the identifier denotes the declared object.   So the set of scopes is 1-1 with the set of declarations.   

We'd need different wording to support what you want.
What you want is tied to the brace-enclosed-block structure of the text.  So we'd have a concept like scope-level which is is tied to a body_statement.

Reply:
Can we just say that declarations in the function parameters or body would shadow the globals?